.github/workflows/django.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         python-version: [3.6, 3.7, 3.8]
         django-version: [2.2.8, 2.2.13, 3.0, 3.1]
-        djangorestframework-version: [3.11.0]
+        djangorestframework-version: [3.11.0, 3.12.0, 3.12.4]
```

Comment:
Why not test only the latest bugfix release in each release branch: 3.11.2 and 3.12.4?